azavea/pfb-network-connectivity
# Feature/kak/places list compare#251

## Overview

Implement comparison feature on places list page.

### Demo

![image](https://cloud.githubusercontent.com/assets/960264/25589078/2cc22150-2e79-11e7-8f6a-942a933a7d94.png)


### Notes

Dropdown list items are currently unstyled.

Also added a footer to the comparison page cards to link to the place detail, since that was in the wireframes.

Refactored some of the comparison place tracking to preserve the offsets of selections, so the cards don't shift around on going back and forth between the place list and comparison pages after removing a selection.


## Testing Instructions

 * Go to [places list](http://localhost:9301/#/places////)
 * Should be able to add and remove places for comparison
 * Shouldn't be able to add the same place twice
 * Should be able to get to comparison page with current selections using last option in compare drop-down
 * Should be able to move between compare and list page with selections preserved

Closes #251
